tests/e2e/basic/wdio.conf.ts

```diff
@@ -8,22 +8,26 @@ const workspacePath = path.resolve(process.cwd(), process.env.VSCODE_E2E_WORKSPA
 export const config: Options.Testrunner = {
     runner: "local",
     specs: ["./specs/**/*.ts"],
-    maxInstances: process.env.RUN_IN_CI ? 1 : 10,
+    maxInstances: 1,
     capabilities: [
         {
             browserName: "vscode",
             browserVersion: process.env.VSCODE_VERSION || "stable",
             "wdio:vscodeOptions": {
                 extensionPath,
                 workspacePath,
+                userSettings: {
```

Comment:
These settings gives me ability to close popups from 2 extensions: npm and git. Which prevent getting data from the log when running tests